src/main/java/ToDo.java

```diff
@@ -1,13 +1,23 @@
 public class ToDo {
     String content;
+    boolean complete;
```

Comment:
These should probably be private (also String content on line 2) as we don't necessarily want them to be immediately accessible outside the class (hence our getter and setter methods).